kolibri/core/assets/src/utils/setupAndLoadFonts.js

```diff
@@ -30,6 +30,9 @@ const modernFontBrowsers = {
     major: 11,
     minor: 4,
   },
+  IE: {
+    major: 100, // specify impossible IE version, forcing passesRequirements to fail
```

Comment:
Hrm, maybe the alternative is to cause `passesRequirements` to return false as the `else` to this clause?
https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/assets/src/utils/browserInfo.js#L21

Reply:
that's what I did at first, until I read that the docstring which says

> It does this in a permissive way, whereby if no specification is made for a particular browser name, then it will return true.

Reply:
Yeah, I think the guy who wrote might have meant it to, because of this use of the same function: https://github.com/learningequality/kolibri/blob/b87f9da274a6a9218419cf9f858be695aa04304a/kolibri/core/assets/src/utils/minimumBrowserRequirements.js#L15

Reply:
this makes sense to handle unknown or strange new browsers

Reply:
I think this is probably fine for now, but we can update the behaviour of `passesRequirements` when we move to using this explicit allowed list of supported browsers: https://github.com/learningequality/kolibri/blob/release-v0.15.x/packages/browserslist-config-kolibri/index.js#L1